userspace/libsinsp/sinsp.h

```diff
@@ -233,7 +236,7 @@ class sinsp_proc_metainfo
   - event retrieval
   - setting capture filters
 */
-class SINSP_PUBLIC sinsp
+class SINSP_PUBLIC sinsp : public capture_stats_source
```

Comment:
The majority of the code that uses sinsp just needs the get_thread function, so it'd be really nice to make a get_thread_source at some point.